tests/e2e/framework/framework.go

```diff
@@ -138,7 +139,11 @@ func (f *Framework) BeforeEach(ctx context.Context) {
 		f.UniqueName = f.Namespace.GetName()
 	} else {
 		// not guaranteed to be unique, but very likely
-		f.UniqueName = fmt.Sprintf("%s-%08x", f.BaseName, rand.Int31())
+		nBig, err := rand.Int(rand.Reader, big.NewInt(31))
+		if err != nil {
+			panic(err)
+		}
+		f.UniqueName = fmt.Sprintf("%s-%08x", f.BaseName, nBig.Int64())
```

Comment:
Question: If we're changing this anyway, why not just use a uuid?

Reply:
I guess the question si - do we want the UUID format here?

Reply:
No longer relevant after addressing your second comment